charts/sn-platform-stack/values.yaml

```diff
@@ -0,0 +1,2336 @@
+#
```

Comment:
This is too big.

If sn-platform or sn-console has a reasonable default, you shouldn't duplicate the value here. The config here should only be the config that sn-platform-stack needs to change. It makes it harder to update the code and keep config in sync otherwise.

Reply:
make sense, will only keep necessary values here